skimage/transform/_geometric.py

```diff
@@ -39,24 +59,30 @@ def _center_and_normalize_points(points):
            (1997): 580-593.
 
     """
-
+    n, d = points.shape
     centroid = np.mean(points, axis=0)
 
-    rms = math.sqrt(np.sum((points - centroid) ** 2) / points.shape[0])
+    centered = points - centroid
+    rms = np.sqrt(np.sum(centered ** 2) / n)
+
+    if rms == 0:
```

Comment:
I guess this point could come pretty close to zero by itself, so is it worth doing the explicit check, or should we just let the algorithm run its course?

Reply:
Or should we define a tolerance parameter for numerical stability?
```python
if rms < tol:
    ...
```

Reply:
`tol` is hard to specify, so I'd say "no, if the algorithm produces nans that pass through anyway" or "yes, if @jni says we need to force the output to have a certain context".  Either way, there may be better ways to check than looking at the RMS---like looking at the condition number of the matrix being solved.

Reply:
OK, after reading more closely, I think that @jni want to avoid these `try... except...` blocks [here](https://github.com/scikit-image/scikit-image/blob/master/skimage/transform/_geometric.py#L306) and [here](https://github.com/scikit-image/scikit-image/blob/master/skimage/transform/_geometric.py#L645).

Reply:
> OK, after reading more closely, I think that @jni wants to avoid these try... except... blocks

bingo! :blush: And those are only triggered when actual 0 is reached. I'll add a comment.